static/src/js/utils/getKmsConceptVersions.js

```diff
@@ -0,0 +1,68 @@
+import xml2js from 'xml2js'
```

Comment:
Can we use fast-xml-parser instead, since we don't have to introduce another library?